config/notifiers.go

```diff
@@ -250,11 +254,47 @@ func (c *SlackAction) UnmarshalYAML(unmarshal func(interface{}) error) error {
 		return fmt.Errorf("missing type in Slack action configuration")
 	}
 	if c.Text == "" {
-		return fmt.Errorf("missing value in Slack text configuration")
+		return fmt.Errorf("missing text value in Slack text configuration")
 	}
-	if c.URL == "" {
-		return fmt.Errorf("missing value in Slack url configuration")
+	// either URL or Name and Value need to be provided
+	if c.URL == "" && c.Name == "" {
+		return fmt.Errorf("missing name or url in Slack action configuration")
 	}
+	if (c.Name == "" || c.Value == "") && c.URL == "" {
+		return fmt.Errorf("missing value in either Slack name and/or value configuration")
+	}
+	return nil
+}
+
+// SlackConfirmationField protect users from destructive actions or particularly distinguished decisions
+// by asking them to confirm their button click one more time.
+// See https://api.slack.com/docs/interactive-message-field-guide#confirmation_fields for more information.
+type SlackConfirmationField struct {
+	Text        string `yaml:"text,omitempty"  json:"text,omitempty"`
+	Title       string `yaml:"title,omitempty"  json:"title,omitempty"`
+	OkText      string `yaml:"ok_text,omitempty"  json:"ok_text,omitempty"`
+	DismissText string `yaml:"dismiss_text,omitempty"  json:"dismiss_text,omitempty"`
+}
+
+// UnmarshalYAML implements the yaml.Unmarshaler interface for SlackConfirmationField
+func (c *SlackConfirmationField) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	type plain SlackConfirmationField
+	if err := unmarshal((*plain)(c)); err != nil {
+		return err
+	}
+	if c.Text == "" {
```

Comment:
Only `text` is required, the other fields are optional: https://api.slack.com/docs/interactive-message-field-guide#confirmation_fields

Reply:
done